### PR TITLE
Java test cases added, site icons set

### PR DESCRIPTION
**Pull request by c-h-i-a-m-a-k-a**

### CODE REVIEW

File: portfolio/src/test/java/com/google/sps/servlets/TestingTest.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import java.util.ArrayList;
```
Comment (mkruskal-google) [inline portfolio/src/test/java/com/google/sps/servlets/TestingTest.java:12]: Move this to a new file (e.g. JournalServletTest)

File: portfolio/src/test/java/com/google/sps/servlets/TestingTest.java [hunk 2]
```diff
@@ -37,11 +38,64 @@ public void testMockito() {
...
+    test.add("string");
+
+    List<String> expected = new ArrayList();
+    expected.add("Unequal amount of subject, type, and salience");
+
+    Assert.assertEquals(expected, JournalServlet.getSubjects(test));
+  }
+  
+  /**
+  @Test
```
Comment (mkruskal-google) [inline portfolio/src/test/java/com/google/sps/servlets/TestingTest.java:78]: Delete all the commented code here and in other files.  With VCS there's no need to keep around old code

File: portfolio/src/test/java/com/google/sps/servlets/JournalServlet.java [hunk 3]
```diff
@@ -1,19 +1,5 @@
-// Copyright 2019 Google LLC
```
Comment (mkruskal-google) [inline portfolio/src/test/java/com/google/sps/servlets/JournalServlet.java:1]: Is your test testing this file or the one in your source code?  You should be testing the same code that's used in production

Comment (c-h-i-a-m-a-k-a) [reply portfolio/src/test/java/com/google/sps/servlets/JournalServlet.java:1]: I've been currently making the code work here, then pasting it into the real file once it pasts the tests, I'm not sure exactly how to call the real functions from TestingTest.java

Comment (mkruskal-google) [reply portfolio/src/test/java/com/google/sps/servlets/JournalServlet.java:1]: You should just be able to import the actual JournalServlet class into your test.  This is fine for debugging purposes, but you shouldn't check this in.  Right now you're only testing a copy of the actual code, not the code itself

**Pull request merged**